### ISSUE TITLE
Infinite loop waiting stack to be ready

### ISSUE THREAD

**Issue by rafaelcaricio** [zalando/lizzy-client#88]
Lizzy client gets in an infinite loop waiting stack that is not in progress to be ready to change traffic.

```
Requesting new stack.. OK
Stack ID: benetton-18
Waiting for new stack... . . . . . . . . . . . . . . . . . . . . . OK
Deployment Successful
Requesting traffic change.. OK
Deleting old stacks..[AGENT] Generating Cloud Formation template..You are deploying an internet-facing ELB that will be publicly accessible! You should have OAUTH2 and HTTPS in place!
[AGENT]  OK
[AGENT] Creating Cloud Formation stack benetton-18.. OK

 > benetton-16 current status is ROLLBACK_COMPLETE trying again later
 OK
Deleting old stacks.. OK

 > benetton-16 current status is ROLLBACK_COMPLETE trying again later
Deleting old stacks.. OK

 > benetton-16 current status is ROLLBACK_COMPLETE trying again later
Deleting old stacks.. OK
```

### Solution
Should wait for stack only if the status is `*_IN_PROGRESS`.

**Comment by marius-oexle** [comment]
I have to extend this a little, encountering the same issue I get an exception at the end
`Stack ID: phoenix-stockupdate-live-green-10
Waiting for new stack... . . . . . . . . . . . . . . . . . . . . . . . . OK
Deployment Successful
Requesting traffic change.. OK
Deleting old stacks..[AGENT] Generating Cloud Formation template.. OK
[AGENT] Creating Cloud Formation stack phoenix-stockupdate-live-green-10.. OK

 > phoenix-stockupdate-live-green-137 current status is UPDATE_IN_PROGRESS trying again later
 OK
Deleting old stacks.. OK

 > phoenix-stockupdate-live-green-137 current status is UPDATE_IN_PROGRESS trying again later
Deleting old stacks.. OK

 > phoenix-stockupdate-live-green-137 current status is UPDATE_IN_PROGRESS trying again later
Deleting old stacks.. OK

 > phoenix-stockupdate-live-green-137 current status is UPDATE_IN_PROGRESS trying again later
Deleting old stacks.. OK

 > phoenix-stockupdate-live-green-137 current status is UPDATE_IN_PROGRESS trying again later
Deleting old stacks.. OK

 > phoenix-stockupdate-live-green-137 current status is UPDATE_IN_PROGRESS trying again later
Deleting old stacks.. OK

 > phoenix-stockupdate-live-green-137 current status is UPDATE_IN_PROGRESS trying again later
Deleting old stacks..
 phoenix-stockupdate-live-green-137
 EXCEPTION OCCURRED: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/lizzy_edge_env/bin/lizzy", line 11, in <module>
    sys.exit(main())
  File "/lizzy_edge_env/lib/python3.5/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/lizzy_edge_env/lib/python3.5/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/lizzy_edge_env/lib/python3.5/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/lizzy_edge_env/lib/python3.5/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/lizzy_edge_env/lib/python3.5/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/lizzy_edge_env/lib/python3.5/site-packages/lizzy_client/cli.py", line 91, in _wrapper
    return func(*args, **kwargs)
  File "/lizzy_edge_env/lib/python3.5/site-packages/lizzy_client/cli.py", line 283, in create
    agent_error(e, fatal=False)
  File "/lizzy_edge_env/lib/python3.5/site-packages/lizzy_client/cli.py", line 72, in agent_error
    data = e.response.json()
  File "/lizzy_edge_env/lib/python3.5/site-packages/requests/models.py", line 826, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/local/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)`

**Comment by rafaelcaricio** [comment]
Fixed by #95
